Gems/AudioSystem/Code/Source/Engine/ATLComponents.cpp

```diff
@@ -1750,34 +1608,20 @@ namespace Audio
     }
 
 
-
-    ///////////////////////////////////////////////////////////////////////////////////////////////////
-    //  SATLSharedData
-    ///////////////////////////////////////////////////////////////////////////////////////////////////
-
-    ///////////////////////////////////////////////////////////////////////////////////////////////////
-    SATLSharedData::SATLSharedData()
-    {
-    }
-
-    ///////////////////////////////////////////////////////////////////////////////////////////////////
-    SATLSharedData::~SATLSharedData()
-    {
-    }
-
-
-
 #if !defined(AUDIO_RELEASE)
 
     ///////////////////////////////////////////////////////////////////////////////////////////////////
-    void CAudioEventManager::DrawDebugInfo(IRenderAuxGeom& rAuxGeom, float fPosX, float fPosY) const
+    void CAudioEventManager::DrawDebugInfo(AzFramework::DebugDisplayRequests& debugDisplay, float fPosX, float fPosY) const
     {
-        static float const fHeaderColor[4] = { 1.0f, 1.0f, 1.0f, 0.9f };
-        static float const fItemPlayingColor[4] = { 0.3f, 0.6f, 0.3f, 0.9f };
-        static float const fItemLoadingColor[4] = { 0.9f, 0.2f, 0.2f, 0.9f };
-        static float const fItemOtherColor[4] = { 0.8f, 0.8f, 0.8f, 0.9f };
+        static const AZ::Color headerColor{ 1.0f, 1.0f, 1.0f, 0.9f };
+        static const AZ::Color itemPlayingColor{ 0.3f, 0.6f, 0.3f, 0.9f };
+        static const AZ::Color itemLoadingColor{ 0.9f, 0.2f, 0.2f, 0.9f };
+        static const AZ::Color itemOtherColor{ 0.8f, 0.8f, 0.8f, 0.9f };
 
-        rAuxGeom.Draw2dLabel(fPosX, fPosY, 1.6f, fHeaderColor, false, "Audio Events [%zu]", m_cActiveAudioEvents.size());
+        const float textSize = 0.8f;
+        AZStd::string str = AZStd::string::format("Audio Events [%zu]", m_cActiveAudioEvents.size());
+        debugDisplay.SetColor(headerColor);
+        debugDisplay.Draw2dTextLabel(fPosX, fPosY, textSize, str.c_str());
```

Comment:
This string can made const as well(You can also switch it over to a `fixed_string<32>` and save the heap allocation.

Reply:
I will update.